config/initializers/secure_headers.rb

```diff
@@ -1,14 +1,14 @@
 SecureHeaders::Configuration.default do |config|
   config.csp = {
     preserve_schemes: true,
-    default_src: %w('none'),
-    connect_src: %w('self' www.google-analytics.com),
-    font_src: %w('self' data:),
-    img_src: %w('self' www.google-analytics.com),
-    manifest_src: %w('self'),
-    media_src: %w('self'),
-    object_src: %w('self'),
-    script_src: %w('unsafe-inline' 'self' www.google-analytics.com),
-    style_src: %w('unsafe-inline' 'self')
+    default_src: ["'none'"],
+    connect_src: ["'self'", "www.google-analytics.com"],
+    font_src: ["'self'", "data:"],
+    img_src: ["'self'", "www.google-analytics.com"],
+    manifest_src: ["'self'"],
+    media_src: ["'self'"],
+    object_src: ["'self'"],
+    script_src: ["'unsafe-inline'", "'self'", "www.google-analytics.com"],
+    style_src: ["'unsafe-inline'", "'self'"]
   }
```

Comment:
This double/single quote stacking looks pretty ugly - could we just tell the linter not to apply that %w rule here instead of re-coding the config?

Reply:
I think we should avoid ignoring files in rubocop, as the config will get out of date as other files change. For me, it's less confusing seeing this than %w('...'), as I can clearly see it's intentional.

Reply:
Fair enough!